src/Console/Command/FixCommand.php

```diff
@@ -184,7 +184,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $progressOutput = $showProgress && $stdErr
-            ? new ProcessOutput($stdErr, $this->eventDispatcher)
+            ? new ProcessOutput($stdErr, $this->eventDispatcher, count($resolver->getFinder()))
```

Comment:
@julienfalque , imagine finder that checks for content, like all files that do not contain `ignore CS`.
but I do agree that this is very edge case.

storing outcome of finder in memory may be difficult if the project is extra large. we could try approach like `if ($count < SOME_LIMIT) { $finder = ArrayIterator(iterator_to_array($finder)); }`
The, all the rest work out of the box, count is cheap and we don't iterate over disc for searching for the files twice if there are not that much files.